Documentation/configuration.md

```diff
@@ -1,34 +1,36 @@
-## Configuration
+# Configuration
```

Comment:
I'm not sure I understand this comment? We should have info on setting kubernetes configuration, but etcd3 support isn't something we have at the moment,